explorer/src/cli/clnodes.ts

```diff
@@ -4,11 +4,11 @@ import {
   ADMIN_USERNAME_HEADER,
   ADMIN_PASSWORD_HEADER,
 } from '../utils/constants'
+import { Config } from '../config'
 
-const EXPLORER_BASE_URL =
-  process.env.EXPLORER_BASE_URL || 'http://localhost:8080'
-const EXPLORER_ADMIN_USERNAME = process.env.EXPLORER_ADMIN_USERNAME
-const EXPLORER_ADMIN_PASSWORD = process.env.EXPLORER_ADMIN_PASSWORD
+const EXPLORER_BASE_URL = Config.baseUrl()
```

Comment:
How do you feel about just calling the function multiple times instead of creating variables now that it's encapsulated?